src/containers/RepositoriesContainer/RepositoriesDucks.js

```diff
@@ -0,0 +1,75 @@
+import ApiUtils from '../../utils/ApiUtils.js';
+
+// Actions
+const LOAD = 'react-redux-starter-app/repositories/LOAD';
+const LOAD_SUCCESS = 'react-redux-starter-app/repositories/LOAD_SUCCESS';
+const LOAD_ERROR = 'react-redux-starter-app/repositories/LOAD_ERROR';
+
+const initialState = {
+  isLoading: false,
+  repos: undefined
```

Comment:
Using it purposefully to differentiate the 3 states:
- Unloaded (undefined)
- Loaded but empty ([])
- Loaded with data